MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -1173,11 +1171,8 @@ private void PlatformDrawPrimitives(PrimitiveType primitiveType, int vertexStart
 
         private void PlatformDrawInstancedPrimitives(PrimitiveType primitiveType, int baseVertex, int startIndex, int primitiveCount, int instanceCount)
         {
-#if GLES || MONOMAC
-            throw new PlatformNotSupportedException("Instanced geometry drawing is not supported yet for GLES or MONOMAC.");
-#else
```

Comment:
@dellis1972 You forgot to remove the matching #endif

Reply:
@Jjagg all fixed up :)